tests/unit/utils/test_yamldumper.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+'''
+    Unit tests for salt.utils.yamldumper
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Salt Libs
+import salt.utils.yamldumper
+
+# Import Salt Testing Libs
+from tests.support.unit import TestCase, skipIf
+from tests.support.mock import patch, NO_MOCK, NO_MOCK_REASON, mock_open
```

Comment:
```suggestion
from tests.support.mock import NO_MOCK, NO_MOCK_REASON
```

According to pylint 😛

Reply:
bah! thanks for that :)

Reply:
BTDT. Too many times 🙃